tests/comments/jsfmt.spec.js

```diff
@@ -1,2 +1,12 @@
 run_spec(__dirname);
 run_spec(__dirname, {parser: 'babylon'});
+
+/**
```

Comment:
please remove all those, I don't think that it's very useful to be in the code. You should keep track of them somewhere else